### PR TITLE
Minor fixes to index page

### PR DESCRIPTION
**Pull request by vpetrusenko**
2018-02-07-160301_1680x1050_scrot
This will appear after user achieve level 3

### CODE REVIEW

File: app/views/index/index.html.erb [hunk 1]
```diff
@@ -8,7 +8,7 @@
 </div>
 
 <div class="row justify-content-center">
-  <div class="col-xs-12 col-sm-11 col-md-6 mt-5">
```
Comment (gfedorenko) [inline app/views/index/index.html.erb]: Why did you delete this class?

Comment (vpetrusenko) [reply app/views/index/index.html.erb]: @gfedorenko i remove only col-md-6 for remove between break

**Comment by gfedorenko** [comment]
Investigate CSS word-break property for fixing #85

**Comment by gfedorenko** [comment]
need to fix this
screenshot from 2018-02-13 14-45-01

**Comment by vpetrusenko** [comment]
@gfedorenko  with last maser i see this
screencapture-localhost-3000-1518528708674

**Comment by gfedorenko** [comment]
@vpetrusenko try resizing

**Comment by vpetrusenko** [comment]
@gfedorenko now it's looks like this
screencapture-localhost-3000-1518598194767
